libs/books/data-access/src/lib/+state/reading-list.selectors.spec.ts

```diff
@@ -1,51 +1,101 @@
-import { initialState, readingListAdapter } from './reading-list.reducer';
+import * as ReadingListActions from './reading-list.actions';
 import {
-  booksAdapter,
-  initialState as booksInitialState
-} from './books.reducer';
-import * as ToReadSelectors from './reading-list.selectors';
+  initialState,
```

Comment:
This test case file is for Selector and you have updated test cases for reducer.
Refrain from doing such changes.